src/test/java/org/broadinstitute/hellbender/tools/coveragemodel/germline/CoverageModellerGermlineSparkToggleIntegrationTest.java

```diff
@@ -0,0 +1,398 @@
+                    TEST_HMM_PRIORS_TABLE_FILE.getAbsolutePath(),
+                "--" + CoverageModellerGermlineSparkToggle.CONTIG_PLOIDY_ANNOTATIONS_TABLE_LONG_NAME,
+                    TEST_CONTIG_PLOIDY_ANNOTATIONS_FILE.getAbsolutePath(),
+                "--" + CoverageModelEMParams.RDD_CHECKPOINTING_PATH_LONG_NAME,
+                    SPARK_CHECKPOINTING_PATH.getAbsolutePath(),
+                "--verbosity", "INFO"
+        }, extraArgs);
+    }
+
+    private String[] getLearningArgs(final String... extraArgs) {
+        return ArrayUtils.addAll(new String[] {
+                "--" + CoverageModellerGermlineSparkToggle.JOB_TYPE_LONG_NAME,
+                    CoverageModellerGermlineSparkToggle.JobType.LEARN_AND_CALL.name(),
+                "--" + CoverageModellerGermlineSparkToggle.INPUT_READ_COUNTS_TABLE_LONG_NAME,
+                    TEST_LEARNING_COMBINED_RAW_READ_COUNTS_FILE.getAbsolutePath(),
+                "--" + CoverageModellerGermlineSparkToggle.SAMPLE_SEX_GENOTYPE_TABLE_LONG_NAME,
+                    TEST_LEARNING_SAMPLE_SEX_GENOTYPES_FILE.getAbsolutePath(),
+                "--" + CoverageModellerGermlineSparkToggle.OUTPUT_PATH_LONG_NAME,
+                    LEARNING_OUTPUT_PATH.getAbsolutePath(),
+                "--" + CoverageModelEMParams.MAX_EM_ITERATIONS_LONG_NAME,
+                    String.valueOf(MAX_LEARNING_EM_ITERATIONS)
+        }, getBaseArgs(extraArgs));
+    }
+
+    private String[] getCallingArgs(final String... extraArgs) {
+        return ArrayUtils.addAll(new String[] {
+                "--" + CoverageModellerGermlineSparkToggle.JOB_TYPE_LONG_NAME,
+                    CoverageModellerGermlineSparkToggle.JobType.CALL_ONLY.name(),
+                "--" + CoverageModellerGermlineSparkToggle.INPUT_READ_COUNTS_TABLE_LONG_NAME,
+                    TEST_CALLING_COMBINED_RAW_READ_COUNTS_FILE.getAbsolutePath(),
+                "--" + CoverageModellerGermlineSparkToggle.SAMPLE_SEX_GENOTYPE_TABLE_LONG_NAME,
+                    TEST_CALLING_SAMPLE_SEX_GENOTYPES_FILE.getAbsolutePath(),
+                "--" + CoverageModellerGermlineSparkToggle.OUTPUT_PATH_LONG_NAME,
+                    CALLING_OUTPUT_PATH.getAbsolutePath(),
+                "--" + CoverageModelEMParams.MAX_EM_ITERATIONS_LONG_NAME,
+                    String.valueOf(MAX_CALLING_EM_ITERATIONS)
+        }, getBaseArgs(extraArgs));
+    }
+
+    private void runLearningAndCallingTest(final String... extraArgs) {
+        runCommandLine(getLearningArgs(extraArgs));
+        final List<Target> modelledTargets = TargetTableReader.readTargetFile(new File(LEARNING_MODEL_OUTPUT_PATH,
+                CoverageModelGlobalConstants.TARGET_LIST_OUTPUT_FILE));
+
+        assertInferredCopyNumberConcordance(LEARNING_POSTERIORS_OUTPUT_PATH, TEST_LEARNING_COMBINED_COPY_NUMBER_FILE,
+                modelledTargets, LEARNING_SEX_GENOTYPES_DATA);
+        logger.info("Copy number concordance test passed for simultaneous learning and calling");
```

Comment:
It's essentially for myself. For the record, this is not a _proper_ integration test. I have no idea how to systematically test all switches. I know what to expect when certain switches are turned on/off, but certainly not all (or their combinations). What is your plan for MuTect integration tests?